.github/ISSUE_TEMPLATE/Bugs.yml

```diff
@@ -0,0 +1,231 @@
+name: Bug report
+description: |
+  Report something that is not working correctly.
+  Not intended for security issues!
+title: Foo.bar should bazoonite, but frobulates instead
+body:
+  - type: markdown
+    attributes:
+      value: "
+        Thank you for taking the effort to report a bug.\n\n
```

Comment:
Would literal style work here? For consistency, I see it's used later

Reply:
No, the heading hashes `##` would not work in that case.